src/Symfony/Component/ExpressionLanguage/ExpressionPhpFunction.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\ExpressionLanguage;
+
+/**
+ * Represents a PHP function that can be used in an expression.
+ *
+ * @author Dany Maillard <danymaillard93b@gmail.com>
+ */
+class ExpressionPhpFunction extends ExpressionFunction
```

Comment:
You can make simple named constructor in `ExpressionFunction` rather than utilizing inheritance:

```php
ExpressionFunction::php('strtoupper')
```